src/tests/java/htsjdk/samtools/util/HistogramTest.java

```diff
@@ -76,6 +78,18 @@ public void testMad() {
     }
 
 
+    @Test(dataProvider = "histogramData")
+    public void testSerializeHistogram(final int[] values, final double unused_mean, final double unused_stdev, final Integer unused_trimByWidth) throws IOException, ClassNotFoundException {
```

Comment:
Yeah, those are weird... They're provided by the data provider, but we don't use them for anything in this tests, so I gave them obvious strange names to let people know.

I'll add a comment stating they're unused and change them to just be `mean`.